test/androidTest/java/org/thoughtcrime/securesms/NicknameEspressoTest.java

```diff
@@ -37,8 +37,37 @@ public void canSetNickname() {
             .goConversation()
             .goSettings()
                 .setNickname(testString)
+                .goConversation()
+            .goConversations()
+            .assertText(testString)
+            .goConversation()
+                .goSettings()
+                .resetNickname()
             .goConversation()
-                .assertText(testString);
+            .goConversations()
+                .assertNoText(testString);
+    }
+
+    @Test
+    public void canSetGroupNickname() {
+        Helper helper = new Helper(mainActivityRule);
+
+        String testString = helper.randString();
+
+        helper
+                .goConversations()
+                .goGroup()
+                    .goSettings()
+                    .setNickname(testString)
+                .goConversation()
+                .goConversations()
+                    .assertText(testString).
+                goGroup(testString)
+                    .goSettings()
+                    .resetNickname()
+                .goConversation()
+                .goConversations()
+                    .assertNoText(testString);
```

Comment:
same with this one, why does it have to reset the nickname? is that part of a bug fix?